ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -626,25 +537,60 @@ export const AdminInstitutionEdit = fp.flow(withNavigation, withRouter)(class ex
     return this.state.institutionMode === InstitutionMode.ADD;
   }
 
+  get isEditInstitutionMode() {
+    return this.state.institutionMode === InstitutionMode.EDIT;
+  }
+
   render() {
     const {institution, showOtherInstitutionTextBox, title} = this.state;
     const {
-      displayName, organizationTypeEnum
+      displayName, organizationTypeEnum, organizationTypeOtherText
     } = institution;
+
+    validate.validators.customEmailAddresses = (accessTierShortName: string) => {
```

Comment:
This is the primary change: moved the email validation checks here.

The Save button now enables/disables instantly (with an informative tooltip) instead of relying on the onBlur() behavior, which can be quite awkward.